lib/binding/diag_kvdb_interface.c

```diff
@@ -21,7 +21,7 @@
 
 #include <hse_util/platform.h>
 #include <hse_util/rest_api.h>
-#include <hse_util/logging.h>
+#include <logging/logging.h>
```

Comment:
We need to qualify our own include files to avoid potential conflict with other include files.  If a subproject happened to have a top-level dir named `logging` that contained a file name `logging.h`, we'd have a build error.  So, how can we make this pattern something like this:
```
#include <hse/logging/logging.h>
```

Reply:
```
mkdir subprojects/xxHash-0.8.1/logging
echo '#error oops' > subprojects/xxHash-0.8.1/logging/logging.h
-->
In file included from ../../../../home/atomlinson/w/src/hse/hse/lib/include/hse_ikvdb/kvdb_rparams.h:19,
                 from ../../../../home/atomlinson/w/src/hse/hse/lib/cn/csched_sp3.c:23:
../../../../home/atomlinson/w/src/hse/hse/subprojects/xxHash-0.8.1/logging/logging.h:1:2: error: #error oops
    1 | #error oops
      |  ^~~~~
```

Reply:
That is pretty doable, but I would like to clean that up in a different PR then if that is cool with you. Then I can tackle pidfile and error at the same time.